src/main/java/nl/novi/techItEasy/services/TelevisionService.java

```diff
@@ -91,6 +74,36 @@ public TelevisionDto getTelevisionById(Long id) {
         }
     }
 
+    public TelevisionDto updateTelevision(Long id, TelevisionDto tvForUpdate) {
+        Optional<Television> t = repos.findById(id);
+        if (t.isEmpty()) {
+            throw new RecordNotFoundException("Television not found");
+        } else {
+            Television tv = t.get();
+            Television tv1 = dtoToTelevision(tvForUpdate);
+            Television returnTelevision = repos.save(tv1);
+            return televisionToDto(returnTelevision);
+        }
+    }
+
+    public void deleteTelevision(@RequestBody Long id) {
```

Comment:
In je service kun je geen @Requestbody annotatie gbruiken. Voordat je een tv delete is het misschien ook handig om te checken of de id waarop je wilt deleten wel aanwezig is in de database (repos.existsById(id))